src/components/PluginWallet/PluginWallet.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import { PluginWallet } from './PluginWallet';
+
+describe('Components | PluginWallet', () => {
+  test('it should render', () => {
+    render(
+      <PluginWallet
+        isActive={true}
+        title={'Massawallet'}
+        iconActive={<></>}
+        iconInactive={<></>}
```

Comment:
done and I add this code in component:
Because linter display an error
```ts
  iconActive: JSX.Element | null;
  iconInactive: JSX.Element | null;
```

Reply:
yes, it's okai now. Thanks